_posts/2018-05-31-regular-types.md

```diff
@@ -0,0 +1,874 @@
+---
+title: "Revisiting Regular Types"
+layout: blog
+sidenav: side-nav-blog.html
+published: true
+permalink: blog/20180531-regular-types
+type: markdown
+category: blog
+excerpt_separator: <!--break-->
+---
+
```

Comment:
Oh, we should maybe add an authorship line here.

Reply:
Done.